### PR TITLE
Feature/jitpack

### PR DESCRIPTION
**Pull request by arturdm**

### CODE REVIEW

File: lib-android/build.gradle [hunk 1]
```diff
@@ -51,19 +53,4 @@ task androidSourcesJar(type: Jar) {
...
-        groupId = 'com.devinci.lib'
-        artifactId = 'android'
-        version = android.defaultConfig.versionName
-      }
-      repository(url: 'file://' + new File(System.getProperty('user.home'),
-          '.m2/repository').absolutePath)
-    }
-  }
-}
+}
```
Comment (tomrozb) [inline lib-android/build.gradle]: It should contain empty line at the end. Do you have "Ensure line feed at file end on Save" enabled in AS?

File: build.gradle [hunk 2]
```diff
@@ -24,8 +25,10 @@ ext {
   configBuildToolsVersion = '23'
   configTargetSdkVersion = 23
 
-  support = [appCompat  : 'com.android.support:appcompat-v7:23.0.0',
-             annotations: 'com.android.support:support-annotations:23.0.0']
+  supportVersion = '23.0.0'
```
Comment (tomrozb) [inline build.gradle:28]: Have you tested it with Crashlytics? I've had problems with versions placeholders when using fabric.io plugin.

Comment (arturdm) [inline build.gradle:28]: you mean with AS fabric plugin or when using this crashlytics module in the app?

Comment (tomrozb) [inline build.gradle:28]: I mean AS fabric plugin. When you add it to the project you cannot build the project when using dependencies with versions set as e.g. $supportVersion. Can you check and confirm?

Comment (arturdm) [inline build.gradle:28]: I tried applying crashlytics to sample app via AS plugin and then switching to this module and it worked fine.

**Pull request merged**